test/src/test/resources/topology/two_standbys_no_lock_holder.properties

```diff
@@ -0,0 +1,16 @@
+# Transport Plugin Configuration
+transport_adapter_JAR_path=../infrastructure/target/infrastructure-0.3.0-SNAPSHOT.jar
+transport_adapter_server_class_name=org.corfudb.infrastructure.logreplication.transport.sample.GRPCLogReplicationServerChannelAdapter
+transport_adapter_client_class_name=org.corfudb.infrastructure.logreplication.transport.sample.GRPCLogReplicationClientChannelAdapter
+
+# Stream Fetcher Plugin Configuration
+stream_fetcher_plugin_JAR_path=../infrastructure/target/infrastructure-0.3.0-SNAPSHOT.jar
+stream_fetcher_plugin_class_name=org.corfudb.infrastructure.logreplication.infrastructure.plugins.DefaultLogReplicationConfigAdapter
+
+# Topology Manager Plugin Configuration
+topology_manager_adapter_JAR_path=../infrastructure/target/infrastructure-0.3.0-SNAPSHOT.jar
+topology_manager_adapter_class_name=org.corfudb.infrastructure.logreplication.infrastructure.plugins.TwoStandbysClusterManager
```

Comment:
Isn't this the same as two_standbys.properties?

Reply:
deleted, I think there were some issues with a netty rpc plugin for this test